### DISCUSSION TITLE
Trouble with Application - Stuck on Processing Information

### DISCUSSION THREAD

**Discussion by winterga** [community/community#102083] [GitHub Education]
### Hi everyone,

I was previously rejected for the Student Developer Pack after submitting an image of an unofficial transcript I have. However, since then, whenever I try to reapply, I get stuck on the "Processing information" page indefinitely. This never happened before, and it occurs regardless of the computer and wifi network I am on. I am trying to submit an image of a student landing page that has my ID and current enrollment status on it. 

How can I fix this? Thank you so much!

**Comment by msuliq** [comment]
Hello, I'm sorry to hear about your difficulties, while I am not in any way related to Github, I can suggest the following actions:

I assume that you have already tried clearing cache, applying in incognito, using different browser, different device and different network, so I will skip these actions. 

You can also check
- that the file is in acceptable format,
- recheck application requirements, in case there were any changes to eligibility criteria,
- if nothing else helps you should contact Github support https://support.github.com/?tags=dotcom-direct, describe your previous actions, your current situation and the way this bug is happening. You might want to send them a screenshot of the "Network" tab of your browser (in developer settings) with an error that shows up after clicking the submit button.

In case you find this answer useful, I would very much appreciate if you could mark it as accepted. Thank you and wish you luck.
